packages/atomic/src/components/ipx/atomic-ipx-modal/atomic-ipx-modal.tsx

```diff
@@ -80,6 +83,10 @@ export class AtomicIPXModal implements InitializableComponent<AnyBindings> {
     this.isOpen && e.preventDefault();
   }
 
+  public componentWillLoad(): void {
+    this.isOpen = this.isOpen || this.isEmbedded;
```

Comment:
Is the embedded version still visually a modal? If not, I recommend removing the `atomic-focus-trap` from the embedded version. This is important because:
* The atomic focus trap prevents keyboard navigation outside of its children.
* The atomic focus trap hides anything outside of it from assistive technology (like screen readers).
* The atomic focus trap has `role="dialog"`, which conveys to assistive technology that there's a dialog currently opened.